conf/modules.config

```diff
@@ -407,3 +407,21 @@ process {
         ]
     }
 }
+
+//
+// PREPARE_MT_ALLIGNMENT
+//
+
+process {
+    withName: '.*PREPARE_MT_ALIGNMENT:SAMTOOLS_VIEW_MT' {
+        ext.args2 = 'chrM'
```

Comment:
This is one of those options that is specific to the genome build. If I remember correctly GRCh37 uses `MT` and hg19 and GRCh38 uses `chrM`. Maybe not much one can do here. @ramprasadn. Do you know if something like this would work?
```groovy
ext.args2 = (params.genome == 'GRCh37') ? 'MT' : 'chrM'
```
Otherwise, don't bother 😄  Just exploring what's possible

Reply:
Good point.. It should work, but the condition might have to be encapsulated within braces like this as it is a closure 😄 

`ext.args2 = { (params.genome == "GRCh37") ? "MT" : "chrM" }`